pkg/gotohelm/transpiler.go

```diff
@@ -699,6 +723,16 @@ func (t *Transpiler) transpileExpr(n ast.Expr) Node {
 			{"_", token.GEQ.String(), "_"}:  f("ge"),
 			{"_", token.LEQ.String(), "_"}:  f("le"),
 
+			{"_", token.EQL.String(), "untyped nil"}: go119eq("eq", "lhs"),
+			{"_", token.NEQ.String(), "untyped nil"}: go119eq("ne", "lhs"),
+			{"untyped nil", token.EQL.String(), "_"}: go119eq("eq", "rhs"),
+			{"untyped nil", token.NEQ.String(), "_"}: go119eq("ne", "rhs"),
+
+			// Support for string + string!
+			{"string", token.ADD.String(), "string"}: func(a, b Node) Node {
+				return builtin("printf", NewLiteral("%s%s"), a, b)
+			},
```

Comment:
It would be good to add one test for that.

Reply:
Done!